transcom/mymove
# B 19137 payment requests billable weight param changes int

## [B-19137](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A922171)

## Summary

Added fuel surcharge to minimum applied billed weight on lookup. The ticket outlines a bunch of service items, though I have found that those appear to already be allowed for payment requests on a shipment with an actual weight under 500 pounds. I tested creating payment requests for them as well, and it was functioning properly. I imagine that these were failing because FSC gets included, and FSC was excluded from the min weight.

[Old pull request](https://github.com/transcom/mymove/pull/12218)

### How to test

1. Login as prime sim
2. Select any move
3. Create a new HHG shipment with an estimated weight of 300
4. As the prime, update the shipment and give it an actual weight of 250
5. Login as TOO -> Approve shipment
6. Login as Prime -> Navigate to move
7. Create payment request on your newly approved HHG shipment for Fuel surcharge (DO NOT PROVIDE A WEIGHT BILLED)
8. It should create successfully with no error
9. Login as TIO -> Navigate to move
10. Your payment request should have a billable weight calculation of “5 cwt” while showing an “original” of 250 lbs, and “estimated” of 300 lbs
11. For thorough testing, repeat steps 6 thru 10 on remaining service items outlined in the ticket’s AC
12. Login as prime -> navigate to your move
13. Create two new service item for your shipment - Domestic origin shuttle service and domestic destination shuttle item. DO NOT SUPPLY WEIGHTS
14. Create a new payment request for DOSHUT without providing a weight billed (It should fail)
15. Create a new payment request for DDSHUT without providing a weight billed (It should fail)
